roles/pulp_devel/templates/cli.settings.toml.j2

```diff
@@ -0,0 +1,6 @@
+[cli]
+base_url = "{{ pulp_webserver_disable_https | default(false) | ternary('http', 'https') }}://{{ ansible_fqdn }}"
+username = "admin"
+password = "{{ pulp_default_admin_password }}"
+verify_ssl = {{ (not pulp_webserver_disable_https | default(true)) | lower }}
```

Comment:
I'm wondering if we should set this to False. In the dev environment we will most certainly always deal with a self signed certificate. Ideally we can push that cert into the local cert storage. But i don't know how.